README.rst

```diff
@@ -134,6 +134,23 @@ If you don't have tox, you can generate a pex of tox:
 
     $ pex tox -c tox -o ~/bin/tox
 
+To run a specific test command, look in `tox.ini` for the name and run like this:
+
+.. code-block::
+
+    $ pex tox -v -e style
+
+Tox allows passthrough arguments, which can be helpful to run specific tests:
+
+.. code-block::
+
+    $ pex tox -v -e py37-integration -- -k test_reproducible_build
```

Comment:
IIRC, I had to modify this line to be able to pass arbitrary passthrough args (`-k test_...`): https://github.com/pantsbuild/pex/blob/master/tox.ini#L44

In particular, I had to change `{posargs:-vvs}` to just `{posargs}`. I may have been doing something else wrong, just checking that this was not the case anymore.